### DISCUSSION TITLE
Whisper UI

### DISCUSSION THREAD

**Discussion by Kuprich** [openai/whisper#1915] [Show and tell]
Hello everyone!

I'm write demo project - [**WhisperUI**](https://github.com/Kuprich/WhisperUIDemo)

You must select audio file, select whisper model (base model by default) and press "recognize" button to start recognition process
![single_tab](https://github.com/Kuprich/WhisperUIDemo/assets/23151696/baf0e904-4e76-4720-986c-d453f02183d1)

Take a look at the process of recognizing multiple files:

![recognition_batch](https://github.com/Kuprich/WhisperUIDemo/assets/23151696/02d3395c-0a40-4426-8a93-38bd27449eb8)

If my project helped someone, please give me a &#9734;, please, because starring a repository also shows appreciation to the repository maintainer for their work.

**Comment by XHD5656123** [comment]
FP16 is not supported on CPU; using FP32 instead
[WinError 2] 系统找不到指定的文件。
这是什么问题呢

**Reply by Kuprich** [reply]
Do you have ffmpeg installed? 
[WinError 2] may appear because ffmpeg is not installed
